apps/client/src/components/randomcarddisplay/randomCardDisplay.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Card, CARD_DECKS, GameMode, Suit } from '@eop/shared';
+import { FC, useEffect, useState } from 'react';
+import DealtCard from '../dealtcard/dealtcard';
+import './randomCardDisplay.css';
```

Comment:
I know, we haven't been doing it so far, but I would suggest to start using CSS modules for new styles, instead of global CSS. In order to do so here, the following changes are needed:
* Rename `randomCardDisplay.css` to `randomCardDisplay.module.css`
* Change this import to `import classes from './randomCardDisplay.module.css';`
* Reference the classes, e.g., instead of `className="card-container"`, we would use `className={classes['card-container']}`

Reply:
done.